definitions/json/CaseEvent/CaseEvent-WA-nonprod.json

```
@@ -0,0 +1,383 @@
+    "EventEnablingCondition": "caseType=\"dummy\"",
+    "CallBackURLAboutToStartEvent": "",
+    "CallBackURLAboutToSubmitEvent": "",
+    "CallBackURLSubmittedEvent": "",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "tseRespond",
+    "Name": "Respond to an application",
+    "Description": "Respond to an application",
+    "DisplayOrder": 62,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "Y",
+    "EventEnablingCondition": "caseType=\"dummy\"",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/tseResponse/aboutToStart",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/tseResponse/aboutToSubmit",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/tseResponse/submitted",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "INITIATE_CASE_DRAFT",
+    "Name": "Create Draft Case",
+    "Description": "Create a new draft case",
+    "DisplayOrder": 3,
+    "PostConditionState": "AWAITING_SUBMISSION_TO_HMCTS",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "SUBMIT_CASE_DRAFT",
+    "Name": "Submit Draft",
+    "Description": "Submit a draft case",
+    "DisplayOrder": 6,
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS",
+    "PostConditionState": "Submitted",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/preDefaultValues",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/postDefaultValues",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/addServiceId",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "preAcceptanceCase",
+    "Name": "Accept/Reject Case",
+    "Description": "Accept/Reject Case",
+    "DisplayOrder": 9,
+    "PreConditionState(s)": "Accepted;Rejected;Vetted",
+    "PostConditionState": "Accepted(preAcceptCase.caseAccepted=\"Yes\"):1;Rejected",
+    "EventEnablingCondition": "caseType !=\"Multiple\" AND preAcceptCase.caseAccepted !=\"Yes\" AND managingOffice !=\"Unassigned\"",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "et3Response",
+    "Name": "ET3 - Respondent Details",
+    "Description": "ET3 - Respondent Details",
+    "DisplayOrder": 43,
+    "PreConditionState(s)": "Accepted",
+    "PostConditionState": "*",
+    "CallBackURLAboutToStartEvent": "${ET_COS_URL}/et3Response/aboutToStart",
+    "CallBackURLAboutToSubmitEvent": "${ET_COS_URL}/et3Response/submitSection",
+    "CallBackURLSubmittedEvent": "${ET_COS_URL}/et3Response/sectionComplete",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Submit ET3 as draft",
```

Any idea on why the label have changed? Should be the below
```suggestion
    "EndButtonLabel": "Save ET3 as draft",
```
Unrelated to the above but for my understanding, why would this event be published? It's a section for the ET3 Form. I would've assumed that this would've been the submitEt3 event.

---

This event was the event requested by the ticket. 
We've received a subsequent CR to update it as you suggest here: https://tools.hmcts.net/jira/browse/RET-4516

---

Cool - as long as there's a ticket to track it, it's fine